react/ui-components/src/molecules/FormStep.js

```
@@ -24,51 +24,86 @@ const FormStep = ({
 
   const goNext = (data) => {
     onSelect(data);
   };
 
-  var isDisable = isDisabled ? true : config.canDisable && Object.keys(errors).filter((i) => errors[i]).length;
+  var isDisable =
+    isDisabled
+      ? true
+      : config.canDisable && Object.keys(errors).filter((i) => errors[i]).length;
 
   const inputs = config.inputs?.map((input, index) => {
```

Biome (2.4.4)

[error] This callback passed to map() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)